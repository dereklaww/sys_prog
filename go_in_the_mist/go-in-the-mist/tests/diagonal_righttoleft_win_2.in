place A5
place A1
place B4
place B1
place C3
place C1
place B2
place B1
place C2
place D2
place D1
place E1