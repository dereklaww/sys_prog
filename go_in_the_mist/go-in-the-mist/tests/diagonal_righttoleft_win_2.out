Black wins!
A5A1B4B1C3C1B2C2D2D1E1
Thank you for playing!